spec/models/contribution_spec.rb

```diff
@@ -12,17 +10,11 @@
     pub
   end
 
-  let(:authorship_json) do
-    # return JSON because this is what the sul-pub API receives
-    pub_with_contrib.pub_hash[:authorship].first.to_json
-  end
-
-  let(:authorship) do
-    # parse the authorship_json because this is what the sul-pub API does
-    JSON.parse(authorship_json)
-  end
+  # return JSON because this is what the sul-pub API receives
+  let(:authorship_json) { pub_with_contrib.pub_hash[:authorship].first.to_json }
+  let(:authorship) { JSON.parse(authorship_json) } # parse authorship_json like sul-pub API
 
-  let(:subject) { pub_with_contrib.contributions.first }
+  subject { pub_with_contrib.contributions.first }
```

Comment:
This comment is not a requirement, but an FYI and optional change; consider http://blog.davidchelimsky.net/blog/2012/05/13/spec-smell-explicit-use-of-subject/; that is, we might call it `contribution` instead of `subject`?  I think this comes up in `reek` analysis.  I stopped using `subject` a while ago, but it's not a rule to use it or not use an explicit subject.  I don't fully understand the smell, but IIRC it has something to do with rspec using the `subject` internally and possible conflicts with that.

Reply:
Yeah, I read that years ago, and I use named subjects where the specs are large and need differentiation.  Specifying `subject` is intended by rspec, and there is no internal conflict presented.   The anti-pattern is overriding `subject` with a `let` statement like the tests currently do:
```ruby
let(:subject) { pub_with_contrib.contributions.first }
```
That is weird.

Reply:
Yep, replacing `let(:subject)` with just `subject` is better, even better is `subject(:name)` whenever any spec needs to explicitly get it's mits on `subject` (I mean `name`).